exe/sufia_export

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env ruby
+require 'rails'
+
+# get the current running directory & load the environment from there
+require ::File.join(Dir.pwd, 'config/environment')
+
+options = {}
+
+args = OptionParser.new do |opts|
+  opts.banner = "Usage: sufia_export [options]"
+
+  opts.on("-v", "--[no-]verbose", "Run verbosely") do |v|
+    options[:verbose] = v
+  end
+
+  opts.on("--limit NUMBER_OF_OBJECTS", Integer, "Export a maximum number of objects (defaults to all)") do |v|
+    options[:limit] = v
+  end
+
+  opts.on("--ids ID_LIST", Array, "Expoprt only the objects in the list of IDs") do |v|
```

Comment:
`s/Expoprt/Export/`

Can you clarify in the help text what `ID_LIST` is supposed to look like? The example you provide for `MODEL_LIST` is a good model